Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -841,6 +841,16 @@
         conditions:
         - !type:ReagentThreshold
           min: 5
+      - !type:RemoveComponent
+        component: ReplacementAccent
+        conditions:
+        - !type:ReagentThreshold
+          min: 5
+      - !type:RemoveComponent
+        component: MonkeyAccent
+        conditions:
+        - !type:ReagentThreshold
+          min: 5
```

Comment:
As above just use comp registry and not multiple effects.

Reply:
I've compressed it down to just one effect with two fields (but using a HashSet instead of a ComponentRegistry)